dev/com.ibm.ws.microprofile.graphql.1.0_fat/fat/src/com/ibm/ws/microprofile/graphql/fat/FATSuite.java

```diff
@@ -47,8 +50,21 @@
                                              .andWith(new FeatureReplacementAction("mpConfig-1.4", "mpConfig-2.0")
                                                       .addFeature("mpMetrics-3.0").removeFeature("mpMetrics-2.3")
                                                       .addFeature("mpRestClient-2.0").removeFeature("mpRestClient-1.4")
-                                                      .withID("mp4.0"));
+                                                      .withID("mp4.0"))
+                                             .andWith(new JakartaEE9Action()
+                                                      .removeFeatures(setOf("mpConfig-1.4", "mpConfig-2.0")).addFeature("mpConfig-3.0")
+                                                      .removeFeatures(setOf("mpMetrics-3.0", "mpMetrics-2.3")).addFeature("mpMetrics-4.0")
+                                                      .removeFeatures(setOf("mpRestClient-2.0", "mpRestClient-1.4")).addFeature("mpRestClient-3.0")
+                                                      .removeFeature("mpGraphQL-1.0").addFeature("mpGraphQL-2.0"));
```

Comment:
Please update and use the MicroProfileActions helper class
https://github.com/OpenLiberty/open-liberty/blob/123a7479dc8b1f9ec8a70de13952e72333074a31/dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java#L272

Reply:
Done